docs/architecture/adr-020-protobuf-transaction-encoding.md

```diff
@@ -83,9 +83,33 @@ message SignerInfo {
         // address can be used for accounts that already have a public key in state 
         bytes address = 2;
     }
-    SignMode mode = 3;
+
+    ModeInfo mode_info = 3;
 }
 
+message ModeInfo {
+    oneof sum {
+        Single single = 1;
+        Multi multi = 2;
+    }   
+   
+    // Single is the mode info for a single signer
+    message Single {
+        SignMode mode = 1;
+        // locale may be used in the future with TEXTUAL signing modes
+        string locale = 2;
```

Comment:
Is this really necessary now? I would rather address specifics for the TEXTUAL mode at a later date when we actually begin fleshing them out.

e.g. If textual mode ends up being something more like a cleaner key-value representation (yaml, etc.) then we wouldn't need locale at all, right?

Reply:
Also, locale is not handled at all in the "multi" mode as this PR is written. Another reason why i think we should drop for now.

Reply:
I just added it so that we don't collapse `Single` into just the `SignMode` enum because we may want other fields describing the mode. I can remove, but `Single` should be a message not just the enum.

And yes it is handled in the multi mode. Study the `Multi` message a bit more